sdk/data/azcosmos/README.md

```diff
@@ -143,6 +143,27 @@ if err != nil {
 itemResponse, err = container.ReplaceItem(context, pk, id, marshalledReplace, nil)
 handle(err)
 
+// Patch an item
```

Comment:
Please also update the CHANGELOG. This would go into the "Features Added" section of the 0.3.3 (Unreleased). With some text like "* Adds Patch support*

Reply:
Done